src/haxelib/api/Vcs.hx

```diff
@@ -323,30 +324,51 @@ class Git extends Vcs {
 	public function clone(libPath:String, url:String, ?branch:String, ?version:String, ?debugLog:(msg:String)->Void):Void {
 		final oldCwd = Sys.getCwd();
 
-		final vcsArgs = ["clone", url, libPath];
+		var vcsArgs = ["clone", url, libPath];
 
-		if (!Vcs.flat)
-			vcsArgs.push('--recursive');
+		Cli.printOptional('Cloning ${name} from ${url}');
```

Comment:
Clone should be updated to take an `optionalLog` argument like `debugLog` rather than calling `Cli.printOptional` directly. This makes the code more reusable for example if someone wants the messages to be displayed in a GUI rather than the cli.